en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+
+To configure a new latency alert, follow the steps below:
+
+1. In the Insights Portal, click **Alert Configuration** to open the **Alert Configuration** page.
+
+    The **Latency** tab is open by default.
+
+2. Verify that you are in the correct organization, and select the required environment. 
+
+3. In the **API Name** field, select the API for which you want to configure the alert.
+
+    !!! Info
+        Only the APIs that you have invoked at least once are listed here. For other APIs, you need to give the API name in the required format as instructed in the UI.
+        
+4. In the **Metric** field, select the required metric against which you want to evaluate the alert configuration.
+
+    !!! tip
+        The list includes all available options. If there are multiple metrics, you can select the required metric. If there is only one metric to select, that metric is selected by default, and the field is disabled. 
+
+5. In the **Latency** field, specify the threshold in milliseconds.
+
+    !!! info
+        When the 95th percentile of the selected metric exceeds the threshold provided here, alerts are triggered.
+        
+6. In the **Emails** field, specify the list of emails that should be notified when the alert is generated.
+        
+7. Click **Add**.
+
+Once an alert is successfully added, the alert configuration is shown in the table as follows. Each configuration can be edited and removed using this table. If multiple emails are configured, they are displayed in a comma-separated list.
+
+#### Traffic alerts
+
+Configure Traffic alerts to identify if the request count of APIs is greater than a predefined threshold. This is useful when you have APIs that are using backends with traffic limits or monetized backends that require your system to proactively scale depending on the incoming traffic.
+
+To configure a new latency alert, follow the steps below:
+
+1. In the Insights Portal, click **Alert Configuration** to open the **Alert Configuration** page.
+
+2. Click **Traffic** to open the **Traffic** tab.
+
+3. Verify that you are in the correct organization, and select the required environment. 
+
+4. In the **API Name** field, select the API for which you want to configure the alert.
+
+    !!! Info
+        Only the APIs that you have invoked at least once are listed here. For other APIs, you need to give the API name in the required format as instructed in the UI.
+        
+5. In the **Metric** field, select the required metric against which you want to evaluate the alert configuration.
+
+    !!! tip
+        The list includes all available options. If there are multiple metrics, you can select the required metric. If there is only one metric to select, that metric is selected by default, and the field is disabled.
```

Comment:
please see the comment on the latency alert section.

Reply:
If there are multiple metrics, you can select only one.  But if only one metric is available, that only available metric is already selected in this field and you can't edit the field.

Reply:
same as above.

Reply:
If the documentation just instructs to select a value, the user might wonder why the field is greyed out and not editable. He/she might not realise it is because only one value is available.